packages/engine-server/src/drivers/file/storev2.ts

```diff
@@ -486,6 +487,15 @@ export class FileStorage implements DStore {
     let notesWithLinks: NoteProps[] = [];
     let errors: IDendronError<any>[] = [];
     const start = process.hrtime();
+    // instantiate so we can use singleton later
```

Comment:
I don't think we should be touching `storev2` anymore to instantiate anything. We should instantiate it during construction of `DendronEngineV3`

Reply:
the reason its here is because we want to gain some confidence with it in our existing workspace. unfortunately `DendronEngineV3` hasn't been implemented yet